apps/webapp/app/send/send-form.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+
+import { useEffect } from 'react';
+import { Button, Switch } from 'ui';
+import { FilledImage, InputBlock, InputToken } from '../../shared';
+import { useStore } from '../../state';
+import { sendSelector } from '../../state/send';
+import { assets } from './constants';
+import { validateAmount, validateRecepient } from '../../utils';
+
+export const SendForm = () => {
+  const {
+    amount,
+    asset,
+    recepient,
+    memo,
+    hidden,
+    validationErrors,
+    setAmount,
+    setAsset,
+    setRecepient,
+    setMemo,
+    setHidden,
+  } = useStore(sendSelector);
+
+  useEffect(() => {
+    // Test logic!!!
+    setAsset(assets[0]!);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
at the moment it should be, here the asset is set at initialisation, in the future we will load the list of assets from the extension and set it in the same way

Reply:
Instead of `Test logic!!!`, think it would be better to add a code comment exactly what you mentioned here. Also, an exclamation point is something that could cause concern. Not necessary here.